easybuild/easyconfigs/a/astro-tulips/astro-tulips-1.0.1-foss-2022a.eb

```diff
@@ -0,0 +1,52 @@
+# Astro-tulips
+
+easyblock = 'PythonBundle'
+
+name = 'astro-tulips'
+version = '1.0.1'
+
+homepage = 'https://astro-tulips.readthedocs.io/en/latest/installation.html'
+description = """tulips creates diagrams of the structure and evolution of stars.
+It creates plots and movies based on output from the MESA stellar evolution code"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('matplotlib', '3.5.2'),
+    ('SciPy-bundle', '2022.05'),
+    ('tqdm', '4.64.0'),
+    ('astropy', '5.1.1'),
+    ('IPython', '8.5.0'),
+    ('FFmpeg', '5.0.1'),
```

Comment:
Other easyconfigs based on `foss/2022a` are using `FFmpeg/4.4.2`.  In order to prevent dependency conflicts, could you stick to that version here as well?